firebase/functions/src/index.ts

```diff
@@ -355,12 +355,14 @@ exports.addProjectTopicKey = functions.https.onRequest(async (_, res) => {
             res.status(404).send('No projects found');
         }
 
-        if (!isEmptyProject && data) {
+        if (data) {
             const newProjectData: {[key: string]: string} = {};
 
             Object.keys(data).forEach((id) => {
-                if (data[id]?.projectTopic) {
-                    const newProjectTopicKey = formatProjectTopic(data[id].projectTopic);
+                const projectData = data[id];
+
+                if (projectData.name) {
```

Comment:
We were previously checking if data[id] is undefined. Now, the check has been removed.

Reply:
we checked data[id].projectTopic not data[id],
Now we little simplified by assigning on the another variable.

Reply:
The analysis is not correct.
The use of `?` means both `data[id]` and `data[id].projectTopic` was checked.